libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -45,6 +45,11 @@ const SKY_DATEPICKER_VALIDATOR = {
   multi: true,
 };
 
+/**
+ * Input directive that provides date input functionality and handles date parsing, formatting and
+ * validation. Emits a `Date` object for valid dates, an empty string when cleared,
```

Comment:
> an empty string when cleared

This is not always true, see the second paragraph in this comment: https://github.com/blackbaud/skyux/pull/3810#discussion_r2269895344